packages/ui/src/theme/tokens/components/badge.ts

```diff
@@ -1,4 +1,44 @@
-export const badge = {
+import {
+  BackgroundColorValue,
+  ColorValue,
+  DesignToken,
+  FontSizeValue,
+  FontWeightValue,
+  LineHeightValue,
+  RadiusValue,
+  SpaceValue,
+  TextAlignValue,
+} from '../types/designToken';
+
+export interface BadgeVariation {
+  backgroundColor: DesignToken<BackgroundColorValue>;
+  color: DesignToken<ColorValue>;
+}
+export interface BadgeSizes {
+  fontSize: DesignToken<FontSizeValue>;
+  paddingHorizontal: DesignToken<SpaceValue>;
+  paddingVertical: DesignToken<SpaceValue>;
+}
```

Comment:
Yea I think this and the alert.ts, we should be consistent. I would say either add `Tokens` at the end or go with a singular name like `BadgeVariation`. I don't have a strong preference because a customer won't use this type explicitly. `BadgeVariationTokens` is a bit more explicit. We could also go with `BadgeVariationTheme`?

Reply:
Updated to follow the same Tokens suffix on all the types.